Userland/Libraries/LibJS/Runtime/GlobalObject.h

```diff
@@ -163,4 +163,17 @@ inline GlobalObject* Shape::global_object() const
 template<>
 inline bool Object::fast_is<GlobalObject>() const { return is_global_object(); }
 
+template<typename... Args>
+[[nodiscard]] ALWAYS_INLINE Value Value::invoke(GlobalObject& global_object, PropertyName const& property_name, Args... args)
+{
+    if constexpr (sizeof...(Args) > 0) {
+        MarkedValueList arglist { global_object.vm().heap() };
+        (..., arglist.append(move(args)));
+        return invoke_internal(global_object, property_name, move(arglist));
+    }
+
+    return invoke_internal(global_object, property_name, Optional<MarkedValueList>{});
+}
+
+
```

Comment:
Not the best location for this but otherwise I ran into dependency cycles.
And since you need to pass GlobalObject you probably have this header included.

Maybe there is some better way? In which case lets change it

Reply:
I don't know of a better way, I think this is fine for now :)